src/main.py

```
@@ -46,16 +55,28 @@
-    # Initialize database tables for both SQLite and PostgreSQL
-    db_config.init_database()
+    try:
+        # Test database connection
+        conn = db_config.get_connection()
+        conn.close()
+        
+        db.create_all()
+        # Initialize database tables for PostgreSQL
+        db_config.init_database()
+        
+        print(f"✓ PostgreSQL database connected successfully")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)